p2-layouts-master/code/index.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Project Name</title>
+    <link rel="stylesheet" href="./style.css" />
+    <!-- dont forget the meta tag for responsiveness -->
+  </head>
+  <body>
+    <!--HEADER-->
+    <header>
+      <!-- If you want a header with a title, logo and a navbar, here is where you should add that. -->
+      <div class="header-text">
+        <h1>Healthy Foood</h1>
+        <h2>Why is healthy food vital.. What are its benefits.. What I love to eat..</h2>
+      </div>  
+    </header>
+
+     <!--NAVIGATOR-->
+    <nav>
+      <a class="nav-link" href="#">Daily Food</a>
+      <a class="nav-link" href="#">Vegan Recipes</a>
+      <a class="nav-link" href="#">Facts & benefits</a>        <a class="nav-link" href="#">Explore The World </a>   
+    </nav>
+
+    <!--CONTENT-->
+    <main class="main-food">
+      <!-- The main container to use as the flex parent for your card layout, the main content of the site should be in here.  -->
+      
+      <div class="food-text">
+        
+        <h3>What I eat in a day</h3>
+
+          <!--PARENT-DAILY-->
+          <div class="daily-container">
+
+              <div class="daily-img">
+                <h4>Breakfast</h4>
+                <img src="b1.jpg" class="daily-img" width="20%">
+              </div>
+
+              <div class="daily-img">
+                <h4>Lunch</h4>
+                <img src="l1.jpg"class="daily-img" width="20%">
+              </div>
+      
+              <div class="daily-img">
+                <h4>Dinner</h4>
+                <img src="d1.jpg" class="daily-img" width="20%">
+              </div>
+      
+              <div class="daily-img">
+                <h4>Snack</h4>
+              <img src="s1.jpg" class="daily-img" width="20%">
+            </div>
+          </div>
+
+        
+        <h3>My favorite vegan recipes</h3>
+
+          <!--PARENT-RECIPES-->
+          <div class="recipes-container">
+
+              <div class="recipes-img">
+                <h4>Vegetables</h4>
+                <img src="veg1.jpg" class="recipes-img" width="20%">
+              </div>
+
+              <div class="recipes-img">
+                <h4>Meat</h4>
+                <img src="mea1.jpg"class="recipes-img" width="20%">
+              </div>
+      
+              <div class="recipes-img">
+                <h4>Legumes</h4>
+                <img src="leg1.jpg" class="recipes-img" width="20%">
+              </div>
+      
+            <div class="recipes-img">
+              <h4>Grains</h4>
+              <img src="gra1.jpg" class="recipes-img" width="20%">
+            </div>
+          </div>
+
+
+        <h3>Fact and benefits of plantbased food</h3>
+    
+          <!--PARENT-FACTS-->
```

Comment:
Would be nice if these parent-facts would be in a container too to be able to create a grid of some sort down here as well :)